Dockerfile

```diff
@@ -2,12 +2,18 @@
 # Use of this source code is governed by an Apache2
 # license that can be found in the LICENSE file.
 
-FROM scratch
+FROM alpine:latest
 
 MAINTAINER Torin Sandall <torinsandall@gmail.com>
 
-ADD /opa_linux_GOARCH /opa
+ENV BIN /usr/bin/opa
 
-ENTRYPOINT ["/opa"]
+RUN apk add --update ca-certificates &&\
+    rm -rf /var/cache/apk/* &&\
+    update-ca-certificates
 
-CMD ["run"]
```

Comment:
As a non-dev of the project I liked it to tell me usage instead of just running. Similar to if I do `curl`, `docker`, `less`, etc. I realize `bash` runs it for you and doesn't give you usage, so again, I could go either way.